dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -464,6 +468,8 @@ public void manualPublish(final List<String> args) {
             tool.setDefaultDockerfilePath(dockerfilePath);
             tool.setDefaultCwlPath(cwlPath);
             tool.setDefaultWdlPath(wdlPath);
+            tool.setDefaultCWLTestParameterFile(testCwlPath);
```

Comment:
So what happens if both values are accidentally the same? 
Alternatively, a user might want to blank the value in the GUI if they know they won't be supporting both languages.